examples/common_vmseries/variables.tf

```diff
@@ -229,40 +273,33 @@ variable "availability_set" {
 
 variable "application_insights" {
   description = <<-EOF
-  A map defining Azure Application Insights. There are three ways to use this variable:
-
-  * when the value is set to `null` (default) no AI is created
-  * when the value is a map containing `name` key (other keys are optional) a single AI instance will be created under the name that is the value of the `name` key
-  * when the value is an empty map or a map w/o the `name` key, an AI instance per each VMSeries VM will be created. All instances will share the same configuration. All instances will have names corresponding to their VM name.
-
-  Names for all AI instances are prefixed with `var.name_prefix`.
+  A map defining Azure Application Insights.
 
-  Properties supported (for details on each property see [modules documentation](../../modules/application_insights/README.md)):
+  For detailed documentation on each property refer to [module documentation](../../modules/application_insights/README.md)
```

Comment:
there are only 4 properties here, I would avoid pointing the user to module's documentation if it's not really necensary, like when you omit some nested props to avoid duplicating documentation.